### DISCUSSION TITLE
Paying for excess Rackspace accounts

### DISCUSSION THREAD

**Discussion by RobertAKARobin** [Geneticure/geneticure.com#62] [General]
@scottsnyder01 Can we cull some of these accounts?

![Screen Shot 2022-10-06 at 8 42 40 AM](https://user-images.githubusercontent.com/6268177/194328715-41802252-5dfb-41e2-a592-83d5f54fa5c0.jpg)

Rackspace Email ($2.40/mo + $1.50/mo for "Email Plus"):
- admin@
- brent@
- info@
- matt@
- support@
- timothy@

Hosted Exchange ($10/mo):
- carl@
- david@
- eric@
- monica@
- robin@
- ryan@
- scott@
- tom@

MS 365 Business Apps ($9.96/mo)
- We have 7 of these, but I can't sign in to MS Admin to see who

**Comment by scottsnyder01** [comment]
We have inactive mailboxes for Matt, Brent and Monica. Working to bring Brent back on soon and should probably leave Matt and Monica for a while in case we need to access old emails.

We have 5 users of one-drive so I eliminated 2 licenses.

From: "Robin (Robert) Thomas" ***@***.***>
Reply-To: "Geneticure/geneticure.com" ***@***.***>
Date: Thursday, October 6, 2022 at 8:43 AM
To: "Geneticure/geneticure.com" ***@***.***>
Cc: Scott Snyder ***@***.***>, Mention ***@***.***>
Subject: [Geneticure] Paying for excess Rackspace accounts (Discussion #62)


@scottsnyder01<https://github.com/scottsnyder01> Can we cull some of these accounts?

[Screen Shot 2022-10-06 at 8 42 40 AM]<https://user-images.githubusercontent.com/6268177/194328715-41802252-5dfb-41e2-a592-83d5f54fa5c0.jpg>

Rackspace Email ($2.40/mo + $1.50/mo for "Email Plus"):

  *   admin@
  *   brent@
  *   info@
  *   matt@
  *   support@
  *   timothy@

Hosted Exchange ($10/mo):

  *   carl@
  *   david@
  *   eric@
  *   monica@
  *   robin@
  *   ryan@
  *   scott@
  *   tom@

MS 365 Business Apps ($9.96/mo)

  *   We have 7 of these, but I can't sign in to MS Admin to see who

**Comment by RobertAKARobin** [comment]
@scottsnyder01 Also in talking to Rackspace support it looks like the "Hosted Exchange" and "365 Business Apps" plans are obsolete, and the cheaper $11.50/mo "365 Business Basic" would give us the same functionality: https://www.rackspace.com/microsoft/office-365/pick-your-plan

If we need offline MS Office apps that's "365 Business Standard" for $13/mo.

With respect to calls/chat, both of those plans include MS Teams which covers both. We might switch to that

**Comment by RobertAKARobin** [comment]
Actually, we could cut out Rackspace altogether and buy directly from Microsoft for $6/mo: https://www.microsoft.com/en-us/microsoft-365/business/compare-all-microsoft-365-business-products-b
